Readme.md

```diff
@@ -40,6 +41,7 @@ Then list your Homebrew based dependencies in your `Brewfile`:
     brew 'mysql'
     cask 'google-chrome'
     cask 'java' unless system '/usr/libexec/java_home --failfast'
+    cask 'firefox', args: {'appdir=/Applications'}
```

Comment:
Ok I'll fix the Readme.
I am stil quite new to ruby and this code base. So I don't know if [this code](https://github.com/Liquidsoul/homebrew-bundle/blob/master/spec/caskinstaller_spec.rb#L9) tests this correctly.